aiven_db_migrate/migrate/pgmigrate.py

```diff
@@ -963,7 +1012,33 @@ def _check_pg_ext(self):
                     dbname
                 )
 
-    def _check_pg_lobs(self):
+    def _check_aiven_pg_security_agent(self) -> None:
+        if not self.target.has_aiven_gatekeeper:
+            return
+
+        self.log.info("Aiven PostgreSQL security agent is enabled in target.")
```

Comment:
should we check `self.target.is_pg_security_agent_enabled` instead? Otherwise the log message is not correct

Reply:
I still think its a bit misleading :sweat_smile: since you are stating that we might do it when it is disabled. It is still saying it is enabled.  Rather have smth like `Aiven PostgreSQL gatekeeper is enabled in target` instead.

Reply:
Hmmm... Did you see the updated message? It's now `self.log.info("Aiven PostgreSQL security agent is used by the target.")`

We should probably resolve that conversation and potentially open a new one on the new line, to avoid any confusion :)

Reply:
That is what I mean. The condition of the if statement does not match with what it is being logged. I understand what you are saying, but the code does not reflect this. So, in a future it might confuse someone else without a previous context

Reply:
Well, maybe I'm missing your point. But if it's in the shared preload libs, that means it's used. Even though it could be enabled or disabled (settings). I think logging "enabled" when it could be disabled would be super confusing, don't you think so?

Reply:
so, my point is that it is a bit confusing that we verify if the target has Aiven Gatekeeper as a setting, and logging that  the security agent is been used. I understand its because of transitive property, but maybe a comment will help others to understand why we are logging that message.

Reply:
@kathia-barahona 
Okay, fair enough. If you feel like it might be confusing, it's valid.
I added a clean docstring to explain this.
Tell me what you think of it.